Author: DoloresPark <dolorespark@e-vertise.com>
Date:   Sat Apr 05 00:52:05 2014 -0500

    enable the 'media.nvidia.audio_alsa' service to be started

diff --git a/cmds/servicemanager/service_manager.c b/cmds/servicemanager/service_manager.c
index 3eaf1eb..701412d 100644
--- a/cmds/servicemanager/service_manager.c
+++ b/cmds/servicemanager/service_manager.c
@@ -32,6 +32,8 @@ static struct {
     { AID_MEDIA, "media.player" },
     { AID_MEDIA, "media.camera" },
     { AID_MEDIA, "media.audio_policy" },
+/* DoPa (20140405) - permit alsa audio service */
+    { AID_MEDIA, "media.nvidia.audio_alsa" },
     { AID_DRM,   "drm.drmManager" },
     { AID_NFC,   "nfc" },
     { AID_BLUETOOTH, "bluetooth" },
